setup.py

```diff
@@ -42,6 +42,9 @@ def unique_flatten_dict(d):
 }
 base_extras_heavy['ai'] = base_extras_heavy['umap-learn'] + ['scipy', 'dgl', 'torch', 'sentence-transformers', 'annoy', 'joblib']
 
+base_extras_heavy['cu-cat'] = + ['cu-cat @ git+https://github.com/graphistry/cuCat/@0.01.0']
```

Comment:
why the `+` ? (is that legal?)

also, let's add `base_extras_heavy['ai'] = ... + base_extras_heavy['cu-cat']` ?